doc/mgr/dashboard.rst

```diff
@@ -249,6 +249,25 @@ into timeouts, then you can set the timeout value to your needs::
 
 The default value is 45 seconds.
 
+Enabling iSCSI Management
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Ceph Manager Dashboard uses the `ceph-iscsi <https://github.com/ceph/ceph-iscsi>`_
+REST API to manage iSCSI.
+
+The available iSCSI gateways must be defined using the following commands::
+
+    $ ceph dashboard iscsi-gateway-list
+    $ ceph dashboard iscsi-gateway-add <gateway_name> <scheme>://<host>[:port]
+    $ ceph dashboard iscsi-gateway-rm <gateway_name>
+
+
+Additionally, the username and password that will be used to access the REST Api,
+can be defined by::
+
+    $ ceph dashboard set-iscsi-api-username <username>     # Defaults to admin
+    $ ceph dashboard set-iscsi-api-password <password>     # Defaults to admin
```

Comment:
Does this mean that each iSCSI gateway needs to use the same username and password here?

If that's the case, wouldn't it be better to embedd the username and password to use into the iSCSI gateway URL defined with the `iscsi-gateway-add` command?